realm/realm-library/src/main/java/io/realm/internal/RealmCore.java

```diff
@@ -115,6 +120,23 @@ private static String loadLibraryWindows() {
         return jnilib;
     }
 
+    private static void loadLibrary(String releaseJniLib, String debugJniLib) {
```

Comment:
We have quite a few `loadLibrary*` methods in this class already. Even though this is a private method it would be nice with a small Java doc comment about what it does that is different from the others.